pkg/api/norman/store/cluster/cluster_store.go

```diff
@@ -609,6 +613,31 @@ func (r *Store) getDynamicField(data map[string]interface{}) (string, error) {
 	return "", nil
 }
 
+func (r *Store) updateClusterByK8sclient(id, name string, data map[string]interface{}) (map[string]interface{}, error) {
+	logrus.Tracef("Updating cluster [%s] using K8s dynamic client", id)
+	object, err := r.ClusterClient.Get(context.TODO(), id, metav1.GetOptions{})
+	if err != nil {
```

Comment:
Are the errors returned here OK for user facing or should we wrap them?

Reply:
What do you mean??  I was getting some errors on testing, and they were showed properly to user facing

Reply:
Updated